chart/jenkins-operator/templates/jenkins.yaml

```diff
@@ -0,0 +1,92 @@
+{{ if .Values.jenkins.enabled }}
+apiVersion: {{ .Values.jenkins.apiVersion }}
+kind: Jenkins
+metadata:
+  name: {{ .Values.jenkins.name }}
+  namespace: {{ .Values.jenkins.namespace }}
+spec:
+  disableCSRFProtection: {{ .Values.jenkins.disableCSRFProtection }}
+  {{- if .Values.backup.enabled }}
+  backup:
+    containerName: {{ .Values.backup.containerName }}
+    action:
+      exec:
+        {{- with .Values.backup.backupCommand }}
+        command: {{ toYaml . | nindent 8 }}
+        {{- end }}
+    interval: {{ .Values.backup.interval }}
+    makeBackupBeforePodDeletion: {{ .Values.backup.makeBackupBeforePodDeletion }}
+  restore:
+    containerName: {{ .Values.backup.containerName }}
+    action:
+      exec:
+        {{- with .Values.backup.restoreCommand }}
+        command: {{ toYaml . | nindent 8 }}
+        {{- end }}
+    {{- if .Values.backup.recoveryOnce }}
+    recoveryOnce: {{ .Values.backup.recoveryOnce }}
+    {{- end }}
+  {{- end }}
+  {{- with .Values.jenkins.plugins }}
+  plugins: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  master:
+    {{- with .Values.jenkins.notifications }}
+    notifications: {{ toYaml . | nindent 4 }}
+    {{- end }}
+    containers:
+      - name: jenkins-master
+        image: {{ .Values.jenkins.image }}
+        imagePullPolicy: Always
+        {{- with .Values.jenkins.imagePullSecrets }}
+        imagePullSecrets: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        livenessProbe:
+          failureThreshold: 12
+          httpGet:
+            path: /login
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 80
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /login
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        {{- with .Values.jenkins.resources }}
+        resources: {{ toYaml . | nindent 10 }}
+        {{- end }}
+      {{- if .Values.backup.enabled }}
+      - name: {{ .Values.backup.containerName }}
+        env:
+          - name: BACKUP_DIR
+            value: {{ .Values.backup.backupDir }}
```

Comment:
env related only to the specific backup provider

Reply:
what do you mean? what is "specific backup provider"